src/main/java/frc/robot/Robot.java

```diff
@@ -89,9 +95,13 @@ public void disabledPeriodic() {}
   /** This autonomous runs the autonomous command selected by your {@link RobotContainer} class. */
   @Override
   public void autonomousInit() {
-    this.trajectoryTutCommand = new TrajectoryTutCommandGroup(drive);
-
-    this.trajectoryTutCommand.schedule();
+    this.path1Command = new AutonPathCommand(drive, "PathWeaver/output/path1.wpilib.json");
+    this.path2Command = new AutonPathCommand(drive, "PathWeaver/output/path2.wpilib.json");
+    this.path3Command = new AutonPathCommand(drive, "PathWeaver/output/path3.wpilib.json");
+    this.drive.resetOdometry(this.path1Command.trajectory.getInitialPose());
+    this.autonCommandGroup = new SequentialCommandGroup(this.path1Command, this.path2Command);
```

Comment:
Commands can be initialized and held onto until they need to be scheduled, and since the `AutonPathCommands` need to read files we want to do that when the robot turns on, not when auto starts. This is for a couple reasons:
  * It saves time when auto starts since the commands have already been created
  * More importantly, if there is an error (like it can't find the file), creating it when the robot turns on will tell us right away instead of waiting until auto starts.

Line 101 can stay where it is, but move the rest into the robot constructor.